checkov/common/bridgecrew/platform_integration.py

```diff
@@ -422,8 +423,13 @@ def persist_enriched_secrets(self, enriched_secrets: list[EnrichedSecret]) -> st
                           f' repo_path={self.repo_path}, bucket={self.bucket}')
             return None
 
-        repo_path_without_src = os.path.dirname(self.repo_path)
-        s3_path = f'{repo_path_without_src}/{checkov_results_prefix}/{CheckType.SECRETS}/secrets_to_verify.json'
+        if not bc_integration.bc_api_key or not os.getenv("CKV_VALIDATE_SECRETS"):
+            logging.debug('Skipping persistence of enriched secrets object as secrets verification is off,'
+                          ' enabled it via env var CKV_VALIDATE_SECRETS and provide an api key')
+            return None
+
+        base_path = self.repo_path.replace('checkov/', 'original_secrets/').replace('/src', '')
```

Comment:
```suggestion
        base_path = self.repo_path.replace('checkov/', 'original_secrets/').replace('/src', '')
```
can you make sure only to replace `checkov/` at the beginning and only remove `/src` at the end. probably easiest to do is with `re.sub()`

Reply:
done :)